src/enochecker/enochecker.py

```diff
@@ -399,6 +399,55 @@ def time_remaining(self) -> int:
 
     # ---- Basic checker functionality ---- #
 
+    def _run_method(
+        self, method: Optional[Union[str, Callable]] = None
+    ) -> Optional[Result]:
+        """
+        Execute a checker method, pass all exceptions along to the calling function.
+
+        :param method: When calling run, you may call a different method than the one passed on Checker creation
+                        using this optional param.
+        :return: the Result code as int, as per the Result enum
+        """
+        if callable(method):
+            return method()
+
+        if method is None:
+            method = self.method
+        if method not in CHECKER_METHODS:
+            raise ValueError(
+                "Method {} not supported! Supported: {}".format(method, CHECKER_METHODS)
+            )
+
+        ignore_run = False
+        if method == "getflag":
+            key = f"__Checker-internals-RESULT:putflag,{self.flag_round},{self.flag_idx}__"
+            if key not in self.team_db:
+                self.info(
+                    f"original putflag did not return successfully -- ignoring getflag for flag_round:{self.flag_round}, index: {self.flag_idx}"
+                )
+                ignore_run = True
```

Comment:
This assignment has no effect since `ignore_run` is overwriten in every case - I think we want to return here

Reply:
good catch, I somehow assumed that there would be an early return when reassigning the `ignore_run` afterwards